Makefile

```diff
@@ -67,17 +67,17 @@ build-coverage: build_cmd=test -c -covermode=count -coverpkg ./pkg/controller/..
 build-coverage: clean $(CMDS)
 
 build-linux: build_cmd=build
-build-linux: arch_flags=GOOS=linux GOARCH=386
+build-linux: arch_flags=GOOS=linux # GOARCH=386
```

Comment:
This was done only because I don't have 32 bit libraries on my system to link to (I think). In theory could be made to work though.